v1/ao/internal/reporter/context.go

```diff
@@ -586,8 +589,11 @@ func NewContext(layer string, reportEntry bool, opts ContextOptions,
 				kvs[k] = v
 			}
 
-			kvs["SampleRate"] = decision.rate
-			kvs["SampleSource"] = decision.source
+			if !continuedTrace {
+				kvs["SampleRate"] = decision.rate
+				kvs["SampleSource"] = decision.source
+			}
+
 			kvs["TokenBucketCapacity"] = fmt.Sprintf("%f", decision.bucketCap)
```

Comment:
Java agent only adds `SampleRate`, `SampleSource`, `BucketCapacity` and `BucketRate` for new trace in service entry  according to https://github.com/librato/trace/blob/3b26fa41c6f3fa9f6472533aa2654ad4d22cd831/docs/specs/KV/service-entry-internal.md#required-for-trace-start-decision

Perhaps we should do the `!continueTrace` check for the token bucket KVs too. Also the KV should be `BucketCapacity` and `BucketRate`